packages/react-native-bpk-theming/stories.js

```diff
@@ -199,12 +196,12 @@ class BpkThemePicker extends Component<{}, State> {
 
     this.themes = {
       blue: generateThemeAttributes(
-        colorBlue400,
-        colorBlue500,
+        colorSkyBlue,
```

Comment:
Given that this is a theming test, shouldn't we pick two different colours to ensure we can see it working?

Reply:
I guess, I removed the gradient because the new brand shouldn't have any gradients anymore.

Reply:
We haven't yet made a decision about that. We might still support it for partners